stand/lua/cli.lua.8

```diff
@@ -24,12 +24,12 @@
 .\" OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
 .\" SUCH DAMAGE.
 .\"
-.Dd July 24, 2021
+.Dd March 29, 2025
 .Dt CLI.LUA 8
 .Os
 .Sh NAME
 .Nm cli.lua
-.Nd FreeBSD Lua CLI module
+.Nd bootloader command line interpreter module
```

Comment:
cli is the module name, though.

Reply:
That's in the name section sir. It renders:
```
cli.lua(8) - bootloader command line interpreter module
```